plugins/modules/proxysql_replication_hostgroups.py

```diff
@@ -33,6 +33,12 @@
     description:
       - Text field that can be used for any purposes defined by the user.
     type: str
+  check_type:
+    description:
+      - Which check type to use when detecting that the node is a slave.
+    type: str
+    choices: [ "read_only", "innodb_read_only", "super_read_only", "read_only|innodb_read_only", "read_only&innodb_read_only" ]
+    default: read_only
```

Comment:
```suggestion
    default: read_only
    version_added: 1.1.0
```

Reply:
Added as 725e02d0ba0779861df45914f9e83f97df5ed4d9

Reply:
@felixfontein added a changelog in be9aa752b4d14421fddee19e0420ad8d58f019ac